bin/plotting/pycbc_plot_dq_flag_likelihood

```diff
@@ -37,24 +40,25 @@ color = pycbc.results.ifo_color(ifo)
 
 fig = pyplot.figure(0)
 ax = fig.add_subplot(111)
-ax.bar(x, height=logrates, label=f'{ifo} DQ Flag Trigger Rates',
+ax2 = ax.twinx()
+
+ax2.bar(x, height=logrates, label=f'{ifo} DQ Flag Trigger Rates',
        color=color)
-ax.set_xlabel('Template Bin Number')
-ax.set_ylabel('Data Quality Log Likelihood')
-ax.set_xticks(x)
-ax.legend(loc='upper left', markerscale=5)
-ax.set_ylim(0, ymax)
-ax.grid()
+ax2.set_ylabel('Data Quality Log Likelihood')
+ax2.legend(loc='upper left', markerscale=5)
+ax2.set_ylim(0, ymax)
+ax2.grid()
 
 yticks = ax.get_yticks()
 
-ax2 = ax.twinx()
-ax2_ymax = numpy.exp(ymax)
-ax2.set_ylim(1, ax2_ymax)
-new_ticks = range(0, int(numpy.ceil(numpy.log10(ax2_ymax))))
-ax2.set_yticks([10**t for t in new_ticks])
-ax2.set_ylabel('Relative Trigger Rate')
-ax2.set_yscale('log')
+ax_ymax = numpy.exp(ymax)
+ax.set_ylim(1, ax_ymax)
+new_ticks = range(0, int(numpy.ceil(numpy.log10(ax_ymax))))
+ax.set_yticks([10**t for t in new_ticks])
+ax.set_ylabel('(Trigger rate during DQ Flag)/(Mean trigger rate)')
```

Comment:
A bit verbose, & would like to incorporate the fact this is truncated to be >=1 somehow.  
```suggestion
ax.set_ylabel('(Rate during DQ Flag)/(Mean Rate) [min 1]')
```